applications/ila-cookie-banner/RELEASE.md

```diff
@@ -0,0 +1,98 @@
+	├── otSDKStub.js
+	├── partials
+	│   └── ila-cookie-banner-content.part.html
+	└── RELEASE.md
+
+### Step 4. Verify the main JavaScript file
+
++ With a text editor, open `otSDKStub.js`
++ For release `1.0.0`, verify that the first line of `otSDKStub.js` contains `1.0.0`:
+	
+	```javascript
+	let cookie_url = 'https://onetrust.techservices.illinois.edu/1.0.0';
+	```
+	
+	> Warning: If the version number (such as `1.0.0`) in the first line does not match the expected release version, the new version will not load properly. Reach out to the development team.
+	
+## Production Release
+
+### Step 5. Activate the New Release
+
++ Make a backup copy of `https://onetrust.techservices.illinois.edu/otSDKStub.js` to `YYYY.MM.DD.otSDKStub.js`
+
+	> Warning: The next step causes the new version of the cookie banner to start appearing across all sites that use the file.
+
++ Copy `otSDKStub.js` from `1.0.0` to the CDN root. 
+ 
+	> Tip: The file `otSDKStub.js` in the CDN root determines which version of the cookie notice is activated.
+
+	> For example: copy `https://onetrust.techservices.illinois.edu/1.0.0/otSDKStub.js` to `https://onetrust.techservices.illinois.edu/otSDKStub.js`
+
+### Step 6. Verify the Release
+
++ In a browser, visit a website that imports `https://onetrust.techservices.illinois.edu/otSDKStub.js` with a `<script src=...>` line. For example, we have been testing with https://cookieme.rslater.web.illinois.edu/
+
++ Press F12 to open the `Developer Tools`.
++ Find and open the `Network` tab of `Developer Tools`.
++ Ensure that `Disable Cache` is checked. (Otherwise the browser may re-use old downloaded files to save time, but break this test.)
++ Reload the page, to capture a copy of all network traffic.
++ Review the rows returned - find the rows from our CDN domain (`onetrust.techservices.illinois.edu`).
+
+	> Tip: If no such rows are found, then the deployment was not fully successful. Follow `How to Roll Back a Release` below, and then reach out to the development team.
+
++ In these rows, confirm that the expected release number (i.e. `1.0.0`) appears in the `GET URL` details. For example:
+
+	```
+	GET https://onetrust.techservices.illinois.edu/1.0.0/partials/ila-cookie-banner-content.part.html
+	```
+
+	```
+	GET https://onetrust.techservices.illinois.edu/1.0.0/css/ila-cookie-banner.css
+	```
+	
+	> Tip: If these URLs *do not include* the expected `onetrust.techservices.illinois` and the current release version (such as `1.0.0`), then the deployment was not fully successful. Follow `How to Roll Back a Release` below, and then reach out to the development team.
+	
+### How to Roll Back a Release
+
+The file `otSDKStub.js` in the CDN root determines which version of the cookie notice is activated.
+
+You can roll back to the previous version by restoring the most recent `YYYY.MM.DD.otSDKStub.js` to `otSDKStub.js` in the CDN root.
+
+Rolling back is recommended if any part of `Step 6: Verify the Release` fails.
```

Comment:
I think this would read better as a single paragraph:

```suggestion
The file `otSDKStub.js` in the CDN root determines which version of the cookie notice is activated. You can roll back to the previous version by restoring the most recent `YYYY.MM.DD.otSDKStub.js` to `otSDKStub.js` in the CDN root. Rolling back is recommended if any part of `Step 6: Verify the Release` fails.
```